adobe-esign-connector/src/com/axonivy/connector/adobe/esign/connector/rest/WebHookService.java

```diff
@@ -55,6 +55,7 @@ public void notification(JsonNode node) {
 		Ivy.log().info(node);
 	}
 	
+	@SuppressWarnings("unused")
 	private boolean verifyClientId(String clientId) {
 		return Optional.ofNullable(Ivy.var().get(CLIENT_ID_VAR)).map(s -> s.compareTo(clientId) == 0).orElse(false);
 	}
```

Comment:
why not just remove the method if it's not used? 
This will be a public product and everybody is going to see it :)